GitExtensions/Project.Loc.targets

```diff
@@ -68,7 +68,7 @@
           ContinueOnError="false" />
 
     <!-- 5. Verify no changes to English*.xlf -->
-    <Exec Command="git add *.xlf --ignore-errors"
+    <Exec Command="git -c core.autocrlf=false add *.xlf --ignore-errors"
```

Comment:
Should it be "autocrlf=input"?

Reply:
Same for the appveyor.yml

Reply:
With `input` the file is not changed when checking out but when adding to index.
In this case, the setting is used when adding to the index, so effectively same as true. 
See comment to mstv above.